README.md

```diff
@@ -594,7 +606,7 @@ Import all exported node SSH keys. Valid values are 'true' and 'false'.
 
 ssh_key_type
 ------------
-Encryption type for SSH key. Valid values are 'rsa', 'dsa', 'ssh-dss' and 'ssh-rsa'
+Encryption type for SSH key. Valid values are 'rsa', 'dsa', 'ssh-dss', 'ssh-rsa', 'ssh-ed25519', 'ed25519', 'ecdsa-sha2-nistp256', 'ecdsa-sha2-nistp384', and 'ecdsa-sha2-nistp521'
```

Comment:
Except that's not entirely true - the ssh_key_type is matched against the currently listed values to determine which host key fact to use - if a new type is added by Puppet labs, using it will cause a compilation failure unless the module is updated to explicitly support it.